php/test/test.php

```diff
@@ -108,6 +110,9 @@ function test_trades($exchange, $symbol) {
 
         dump(green($symbol), 'fetching trades...');
         $trades = $exchange->fetch_trades($symbol);
+        if (count($trades) > 0) {
+            testTrade ($exchange, $trades[0], $symbol, time() * 1000);
```

Comment:
Ideally, we want to follow a uniform codestyle everywhere:

- camelcase for JS
- snake notation for Python
- snake notation for PHP